pkg/rpmdb_test.go

```diff
@@ -119,6 +119,17 @@ func TestPackageList(t *testing.T) {
 			}
 		})
 	}
+
+	for _, tt := range tests {
```

Comment:
Do we need this test? `ListPackages` is already tested above and this test doesn't seem to add any value for testing. `BenchmarkXXX` makes more sense.

Reply:
Definitely. Let me arrange that.

Reply:
Done 24028bbccbd2b1dc6a530706599e072c6aaff512